RELEASE_NOTES

```diff
@@ -1,3 +1,7 @@
+Version 2.4.0
+-------------
+59. WhatsApp: update supported_version annotation to 2.24.23.78
```

Comment:
It does look like a little bit more happened than just updating the version annotation. Should we add something in the release notes about the updated WhatsApp actions?